nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-client-service-api/src/main/java/org/apache/nifi/mongodb/MongoDBClientService.java

```diff
@@ -40,33 +41,33 @@ public interface MongoDBClientService extends ControllerService {
      String WRITE_CONCERN_W2 = "W2";
      String WRITE_CONCERN_W3 = "W3";
 
-     static final AllowableValue WRITE_CONCERN_ACKNOWLEDGED_VALUE = new AllowableValue(
```

Comment:
Is there a reason that these shouldn't be `static final`? Otherwise this LGTM

Reply:
This is a helpful improvement, `static final` is implicit on members of an interface definition.

Reply:
IntelliJ said that was redundant, so I removed it.